test/fixtures/codemod/latest-node/my-app/app/controllers/application.js

```diff
@@ -11,6 +11,10 @@ export default Controller.extend({
     return `${this.firstName} ${this.lastName}`;
   },
 
+  friendNames: map('friends', ['nameKey'], function(friend) {
```

Comment:
Can you import `map`? Just in case the codemod starts following the source in the future, and not codemodding things that don't come from ember.

Reply:
Sure, similarly for the computed properties also we need to import the computed right?

Reply:
Probably, but I wouldn't worry about it for this pull request

Reply:
Ok then, I will make changes for importing map